core/refinement/refined_class_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../../spec_helper'
+
+describe "Refinement#refined_class" do
+  ruby_version_is "3.2" do
+    it "returns the class refined by the receiver" do
+      r = Module.new do
+        refine Integer do
+          refined_integer = self
+        end
+
+        refine String do
+          refined_string = self
+        end
+      end.refinements
+
+      r[0].refined_class.should == Integer
+      r[1].refined_class.should == String
+    end
```

Comment:
**suggestion:** I would avoid using `#refinements` here. It isn't necessary and makes a test more _integrating_.

Also it's enough only one refinement in the test.